plaaces-to-go/src/styles/Menu.module.css

```diff
@@ -0,0 +1,101 @@
+.navbar{
+    display: flex;
+    align-items: center;
+    background: #f7e9da;
+}
+
+.navLogo{
+    display: flex;
+    flex: 1;
+}
+
+.txtItem{
+    color: #3f6b7c;
+    font-size: 20px;
+    padding: 20px;
+    font-family: 'Lato', 'Arial Narrow', Arial, sans-serif;
+    font-weight: 800;
+}
+
+.navItems{
+    display: flex;
+    flex-direction: row;
+    justify-content: flex-end;
+    align-items: center;
+    flex: 2;
+    color: #71899b;
+    text-decoration: none;
+    list-style-type: none;
+    padding: 0 20px;
+    font-family: 'Lato', 'Arial Narrow', Arial, sans-serif;
+    font-weight: 700;
+    margin: 0;
+}
+
+.navItems li:hover{
+    color: #b97479;;
+}
+.navItems li{
+    padding: 0 15px;
+}
+
+.navToggle {
+    display: none;
+}
+
+@media (max-width: 768px) {
+    .navItems{
+        position: absolute;
+        top: 60px;
+        left: 0;
+        background:#f7e9da;
+        display: flex;
+        flex-direction: column;
+        width: 100%;
+        height: -webkit-fill-available;
+        transition: 0.3s ease all;
+        align-items: initial;
+        justify-content: flex-start;
+        transform: translateX(-100%);
+    }
+
+    .navItems.openNav{
+        transform: translateX(0)!important;
+    }
+
+    .navToggle{
+        display: flex!important;
```

Comment:
We need to figure out a way to make this work without the need for the `!important`s

Reply:
I remove all the important :P